framework/python/src/api/api.py

```diff
@@ -168,7 +168,21 @@ async def post_sys_config(self, request: Request, response: Response):
     try:
       config = (await request.body()).decode("UTF-8")
       config_json = json.loads(config)
+
+            # Validate req fields

```

Comment:
This comment should not be indented this much

Reply:
Solved